oracle.go

```diff
@@ -27,12 +27,15 @@ DROP USER {{username}};
 `
 
 	defaultRotateCredsSql = `ALTER USER {{username}} IDENTIFIED BY "{{password}}"`
+
+	defaultUsernameTemplate = `{{ printf "V_%s_%s_%s_%s" (.DisplayName | uppercase | truncate 8) (.RoleName | uppercase | truncate 8) (random 20 | uppercase) (unix_time) | truncate 30 }}`
```

Comment:
Nit: Is there a way to call `uppercase`  just once? Maybe `(.DisplayName | truncate 8) (.RoleName | truncate 8) (random 20) (unix_time) | uppercase | truncate 30`?

Reply:
Yes, that will work. Good catch. I've also added the dashes & periods to underscore into the template instead of in the code. This will allow users to specify dashes and periods if they want to, but it will be up to to them to ensure that the statements are formatted correctly to handle those characters.